spec/inertia/sharing_spec.rb

```diff
@@ -54,63 +54,6 @@
     it { is_expected.to eq props.merge({ errors: errors }) }
   end
 
-  context 'multithreaded inertia share' do
```

Comment:
@ElMassimo just for curiosity, why do you remove this test? It was failing or because we do not have this problem anymore by default?

Reply:
It's not possible to have multi-threading issues anymore, as the objects that can be shared between threads are now frozen (not writable, and thus not susceptible to concurrent writes either).

Reply:
Thanks for the response!!

Reply:
Fwiw, i pulled the branch and added these tests back, and they do still pass.

These were made unnecessary by #111. At the time, I asked @PedroAugustoRamalhoDuarte to keep the specs so we weren't changing spec + implementation at the same time. I'm good with removing them now that we have evidence that refactor passed the original spec. (kind of like adding a new DB column, migrating data, then removing the old column once everything is verified working in production instead of just renaming the column in one shot)